<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cashbox</class>
 <widget class="QDialog" name="cashbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="product_name">
     <item>
      <property name="text">
       <string>----------------------</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="shop_price">
     <property name="placeholderText">
      <string>Цена в магазине</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="vat">
     <property name="placeholderText">
      <string>НДС</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="bank_name">
     <property name="placeholderText">
      <string>Банк</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
